solvcon/dependency.py

```diff
@@ -64,8 +75,10 @@ def import_module_may_fail(modname, asname=None):
     cframe = inspect.currentframe().f_back # Caller's frame.
     try:
         cglobals = cframe.f_globals
-        # Determine package name for importlib.import_module().
-        if modname.startswith('.'):
+        if _libmarch_name_pattern.search(modname):
```

Comment:
With this, I don't need to change other places in the Python code to import libmarch instead of solvcon.march.